api/execution/src/main/java/quarks/execution/mbeans/PeriodicMXBean.java

```diff
@@ -16,14 +16,14 @@ Licensed to the Apache Software Foundation (ASF) under one
 specific language governing permissions and limitations
 under the License.
 */
-package quarks.oplet.core.mbeans;
+package quarks.execution.mbeans;
 
 import java.util.concurrent.TimeUnit;
 
 /**
- * Control interface for a periodic oplet.
+ * Control interface for a periodic source stream.
```

Comment:
Since this has moved it now also seems completely generic it could be used for periodic entity, therefore:

`s/source stream/entity/`

?